### PR TITLE
Version bump 1.3.1.dev and CI config updates

### PR DESCRIPTION
**Pull request by bmbouter**
* Now at 1.3.1.dev in prep for next z-stream release
* Declared compatability with pulpcore 3.7.0 in CI variables

[noissue]

### CODE REVIEW

File: template_config.yml [hunk 1]
```diff
@@ -38,8 +38,8 @@ pulp_settings:
   - sha256
   - sha384
   - sha512
-pulpcore_branch: master
-pulpcore_pip_version_specifier: null
+pulpcore_branch: 3.7
+pulpcore_pip_version_specifier: ~=3.7.0
```
Comment (mdellweg) [inline template_config.yml:42]: Doesn't this need regenerating the CI stuff with plugin_template?

File: template_config.yml [hunk 2]
```diff
@@ -38,8 +38,8 @@ pulp_settings:
   - sha256
   - sha384
   - sha512
-pulpcore_branch: master
-pulpcore_pip_version_specifier: null
+pulpcore_branch: 3.7
```
Comment (ipanova) [inline template_config.yml:41]: can you also update stable_branch?

File: template_config.yml (hunk 1 above)
Comment (bmbouter) [reply template_config.yml:42]: I'm not sure. I got this step from the [y-release checklist](https://pulp.plan.io/projects/pulp/wiki/Pulp3_Release_Guide#Preparing-a-Y-release) so I don't know what I'm doing. Can someone from the CI team give some advice here?

File: template_config.yml (hunk 2 above)
Comment (bmbouter) [reply template_config.yml:41]: I can, but I'm not sure what I'm doing. I got this step from the [y-release checklist](https://pulp.plan.io/projects/pulp/wiki/Pulp3_Release_Guide#Preparing-a-Y-release) so I don't know what I'm doing. Can someone from the CI team give some advice here?

Comment (ipanova) [reply template_config.yml:41]: it is step 7 from the checklist

Comment (ipanova) [reply template_config.yml:41]: as i figured out this is used fort he cherrypick automation that is disabled for this plugin, we can ignore this variable

File: template_config.yml (hunk 1 above)
Comment (mdellweg) [reply template_config.yml:42]: This file is driving the templating in pulp/plugin_template so every time that file is changed, i expect `./plugin-template --travis pulpcore` to be run.

Comment (daviddavis) [reply template_config.yml:42]: I believe @mdellweg is correct although it's `./plugin-template --travis pulp_file`.

Comment (daviddavis) [reply template_config.yml:42]: I'll update the release guide and checklist.

Comment (mdellweg) [reply template_config.yml:42]: I think, we should add a not _in_ the `template_config.yml`

**Comment by bmbouter** [comment]
@ipanova @mdellweg I pushed new changes, wdyt about these? I was a little confused on the template_config changes, so i based those on [this other PR](https://github.com/pulp/pulp_container/pull/159/files) as an example. I also regenerated the template.

**Review by ipanova — APPROVED** [review]

File: template_config.yml [hunk 3]
```diff
@@ -38,13 +38,13 @@ pulp_settings:
...
+pulpcore_pip_version_specifier: ">=3.7,<3.9"
 pulpprojectdotorg_key_id: aa499d7938ed
 pydocstyle: true
 pypi_username: pulp
 redmine_project: pulp_file
-stable_branch: '1.2'
+stable_branch: None
```
Comment (bmbouter) [inline template_config.yml:47]: I need to change this to null actually I think.

File: template_config.yml (hunk 1 above)
Comment (mdellweg) [reply template_config.yml]: ^ "... add a note, ..."

Comment (bmbouter) [reply template_config.yml]: What sort of note? I didn't understand the suggestion.

**Pull request merged**